docs/modules/ROOT/pages/index.adoc

```diff
@@ -137,22 +138,39 @@ nile counterfactual-address ACCOUNT_1 --salt 123
 +
 NOTE: Different salts compute to different addresses, even with the same private key.
 +
-. In production, you will need to fund this address with Ether to pay for the deployment fees, which will be mandatory in testnets soon. There are multiple bridges and faucets, like link:https://goerli.starkgate.starknet.io/[Starkgate] being implemented at the moment to support the developer experience.
+. Estimate the fee for executing the transaction.
 +
-For local development using a devnet node, you can use predeployed accounts to fund your address by using xref:scripts.adoc#transfer_funds_from_predeployed_devnet_account[this script].
+[,sh]
+----
+nile setup ACCOUNT_1 --salt 123 --estimate_fee
+----
++
+[,sh]
+----
+The estimated fee is: 52147932632842 WEI (0.000052 ETH).
+Gas usage: 5063
+Gas price: 10299808934 WEI
+----
```

Comment:
I think this can be a good place to let the user know about the `--estimate_fee` feature, even if we mention below that Nile will automatically set the max_fee if not passed, because users need to pre-fund the counterfactual-address, and this gives them the amount they need to add. What do you think?

Reply:
Sounds good to me!